src/views/Login.vue

```diff
@@ -21,7 +22,7 @@ export default class Login extends Vue {
             requestedKeyPaths: [`m/44'/242'/0'/0'`],
         };
 
-        const client = this.$rpc.createKeyguardClient();
+        const client = this.$rpc.createKeyguardClient(Config.keyguardEndpoint);
```

Comment:
Wouldn't it make more sense to set this in the `RpcApi` class on the method itself, instead of having to set it in every request?

Reply:
Definetly.